model_compression_toolkit/gptq/keras/quantizer/gumbel_rounding/base_gumbel_rounding.py

```diff
@@ -31,21 +31,29 @@
 P_INIT = 0.01
 
 
-def _init_aux_var(w_shape: List[int], m: int, p: float = P_INIT) -> np.ndarray:
+def init_aux_var(ceil_indicator: np.ndarray, w_shape: List[int], m: int, p: float = P_INIT) -> np.ndarray:
     """
     This function generate a random pi matrix for Gumbel Rounding
     Args:
-        w_shape(List[int]): A list of integers that represent the shape of the weights tensor to be quantization
+        ceil_indicator: an array of indicator if the value should be ceil or floor.
```

Comment:
Really picky but if you're doing another iteration on this file then we might as well clean this.
"**A**n array of indicator**s**..."